src/vorta/application.py

```diff
@@ -100,12 +102,21 @@ def toggle_main_window_visibility(self):
 
     def backup_started_event_response(self):
         self.tray.set_tray_icon(active=True)
+        self.tray.tooltip_tray_txt(active=True)
+
+    def set_tray_tooltip(self):
+        if BorgThread.is_running():
+            self.tray.tooltip_tray_txt(active=True)
+        else:
+            self.tray.tooltip_tray_txt(active=False)
```

Comment:
This can be replaced with an "active" variable in TrayMenu

Reply:
No it can not, we need this if the task is running and you click "Apply" on schedule tab it should still active.